trimmer_decorator.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    name = @nameable.correct_name
+    return name[0..9] if name.length > 10
+
+    name
+  end
```

Comment:
[OPTIONAL] You can implement this method in the following manner to save you some lines of code.

<img width="733" alt="Screen Shot 2022-08-23 at 9 11 15" src="https://user-images.githubusercontent.com/82426682/186180635-4c0fa3e1-a184-4d44-8aae-5a18c300f7e7.png">

Reply:
What elegance the one of France 👍